README.md

```diff
@@ -145,24 +150,26 @@
 
 ## 👥 Authors <a name="authors"></a>
 
-👤 **Daniel Villalba Lynch**
+👤 **Daniel Villalba**
 
```

Comment:
- [ ] According to GitHub history 3 persons contributed to this project, but we have only one person information under the **Authors** section. Kindly make sure we have all the authors listed in the **Authors** section to make your README file looks more professional.